src/azure_keyvault_browser/__init__.py

```diff
@@ -1,7 +1,14 @@
-import importlib.metadata
 import sys
 
-if (sys.version_info[0], sys.version_info[1]) < (3, 9):
-    sys.exit("keyvault-browser requires Python 3.9 or later.")
+if (sys.version_info[0], sys.version_info[1]) < (3, 7):
+    sys.exit("azurecr-browser requires Python 3.7 or later.")
+
+try:
+    import importlib.metadata
+
+    __version__ = importlib.metadata.version("azure_keyvault_browser")
+except ModuleNotFoundError:
+    import importlib_metadata
```

Comment:
This looks duplicated. Importlib was added in 3.8 so should we be falling back to something else here for 3.7 support?

Reply:
Yep - that's the goal. As you say, `importlib` is available in 3.8, but not in 3.7. The proposal here is to fall back to the [importlib-metadata](https://importlib-metadata.readthedocs.io/en/latest/) package (a backport of the core functionality of `importlib`), when Python 3.7 is being used.

Reply:
oh my bad i completely missed the module name!

Reply:
It makes sense, I guess, that they named the module that way... but really does not make it easy to differentiate 😀